README.md

```diff
@@ -4,8 +4,9 @@ cli to serve polyglot monorepo
 
 ## Features
 
-- [ ] Ability to execute "scripts" across monorepo: start, build, test, fmt, etc - `mrt run [script]`
-- [ ] Support for nodejs/npm packages and apps
+- [x] List package in the monorepo - `mrt list`
+- [x] Ability to execute "scripts" across monorepo: start, build, test, fmt, etc - `mrt run [script]`
```

Comment:
Isn't it too early to set checkmark here? You have at max prototype for this. My assumption that you want to have common linguo for all projects independently from how operation called in specific package manager. For example `fmt` in one, `format` in another. Most likely you would need to preserve language native names in the proejct definitions.

Even if you have common lingo across different project types, then you probably somehow reflect this as expectation.

Reply:
You are right regarding common linguo, however I want this to be checked as it already working. Such normalization of scripts would be probably done in concrete implementations, like for python_porety.

Reply:
I assuming that you want consistent way to run operations on each project, but that means that you need spec for each command. Based on that premise, I was thinking that maybe it’s too early to check marks for “all” commands. But hey, that’s your ToDo list so do as you deems nescessary. Nothing specific here

Reply:
I am considering this feature as a dumb script runner, and later will probably attempt to implement som magic.

Reply:
Yeah. Make sense.